### PR TITLE
added recursive option

### PR DESCRIPTION
**Pull request by ginman86**

### CODE REVIEW

File: example/resources/users/history/index.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+'use strict'
+
+module.exports = function (app) {
+  /**
+   * route handler for /history.
```
Comment (wilmoore) [inline example/resources/users/history/index.js]: Would we want this comment to be `/users/history`?

File: index.js [hunk 2]
```diff
@@ -29,8 +29,8 @@ module.exports = packages
  * KOA application instance.
  */
 
-function packages (dir, arg, skip) {
-  var modules = find.in(dir).findDirectories()
+function packages (dir, arg, skip, recursive) {
+  var modules = recursive ? find.from(dir).findDirectories() : find.in(dir).findDirectories()
```
Comment (wilmoore) [inline index.js]: Let's just make this the default and drop the `recursive` option. No reason not to make it recursive AFAIKT. If you can think of any issues with that, let me know, but seems fine.

**Comment by wilmoore** [comment]
Would you mind squashing your commits? This means being familiar with [interactive rebasing](https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase-i).

**Comment by ginman86** [comment]
squarshd

**Comment by wilmoore** [comment]
LGTM...thanks :+1:

**Pull request merged**